File: C#/Isomorphic Strings/Isomorphic Strings.cs
public class Solution {
    public bool IsIsomorphic (string s, string t) {
        if (s.Length <= 1) return true;
        Dictionary<char, char> table = new Dictionary<char, char> ();
        for (int i = 0; i < s.Length; i++) {
            if (table.Keys.Contains (s[i])) {
                if (table[s[i]] != t[i]) return false;
            } else {
                if (table.Values.Contains (t[i])) return false;
                table.Add (s[i], t[i]);
            }
        }
        return true;
    }
}

File: Python/Isomorphic Strings/Isomorphic Strings.py
class Solution:
    def isIsomorphic(self, s: str, t: str) -> bool:
        hmap = {}
        for i in range(len(s)):
            if(s[i] in hmap.keys()):
                if(hmap[s[i]] != t[i]):
                    return False
            else:
                if(t[i] in hmap.values()):
                    return False
                hmap[s[i]] = t[i]
        return True
